tools/config.py

```diff
@@ -28,6 +29,24 @@ def read_file(path):
         error(f'Unable to read configuration file {path}!')
 
 
+def read_config(path='app.cfg'):
+    """Read the config file
+    Args:
+        path (string): path to the configuration file
+    Returns:
+        dict (str, dict): dictionary containing configuration settings
+    """
+    fn = sys._getframe().f_code.co_name
+
+    try:
+        config = configparser.ConfigParser()
+        config.read(path)
+    except Exception:
+        error(f"{fn}(): Unable to read configuration file {path}!")
+
+    return config
+
+
```

Comment:
Has this leaked into the PR from #130?

Reply:
After #130 is merged pull in changes into development branch (dedicated_log). Maybe conflicts need to be resolved.